clients/linux/src/app.rs

```diff
@@ -207,7 +220,10 @@ impl LbApp {
         // used to receive progress updates as indicated above.
         spawn!(self.core as c, self.messenger as m => move || {
             if let Err(err) = c.sync(sync_chan) {
-                m.send_err("syncing", err);
+                match err.target() {
+                    LbErrTarget::Dialog => m.send_err_dialog("syncing", err),
```

Comment:
It depends on the specified `target` of the error. If it is `LbErrTarget::Dialog`, then it will show up in a dialog, and if it is the `LbErrTarget::StatusPanel`, it will show up in the status panel.